tmcprototype/SdpMasterLeafNode/test/SdpMasterLeafNode_test.py

```diff
@@ -52,7 +52,7 @@ class TestSdpMasterLeafNode(object):
     device = SdpMasterLeafNode
     properties = {'SkaLevel': '4', 'GroupDefinitions': '', 'CentralLoggingTarget': '',
                   'ElementLoggingTarget': '', 'StorageLoggingTarget': 'localhost',
-                  'SdpMasterFQDN': 'mid-sdp/elt/master',
+                  'SdpMasterFQDN': "mid-sdp/elt/master"
```

Comment:
what is the reason to keep this single value in double-quotes when all other values are in the single quote? Check if it can be in single quotes for consistency.

Reply:
Single quote mark will be followed everywhere. By mistake it is different for SdpMasterFQDN.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd